src/book/03-guides/02-tools/03-webots.mdx

```diff
@@ -96,25 +96,27 @@ sudo apt-get install cmake-curses-gui libprotobuf-dev protobuf-compiler libeigen
    cd NUWebots
    ```
 
-2. Compile the controllers with cmake
+2. Configure the codebase by running
 
-   1. Make a build folder and move into it
-
-      ```sh
-      mkdir build && cd build
-      ```
+   ```sh
+   ./b configure [-i] [-- <flags>]
```

Comment:
This might be giving people too many choices in a starting guide - it's still good to have, but maybe make a note that if you're not sure what to run just run `./b configure`.

Reply:
I might make a `details` with those details

Reply:
I think that's a lot nicer